### PR TITLE
add openSUSE-repos CDN article

### PR DESCRIPTION
**Pull request by lkocman**
Do not merge until new TW snapshot is published (eta beginning of next week)
Review of the text is highly welcome.

screen

### CODE REVIEW

File: _posts/2023-07-25-try-out-cdn-with-opensuse-repos.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- MicroOS
+- Leap
+- zypper
+- RIS
+
+---
+
+[openSUSE-repos](https://github.com/openSUSE/openSUSE-repos) simplifies openSUSE repository management by utilizing [Repository Index Service (RIS)](https://en.opensuse.org/openSUSE:Standards_Repository_Index_Service) for core distribution repositories.
+
+The latest update switches users, who already use openSUSE-repos, to the new [cdn.opensuse.org](https://code.opensuse.org/leap/features/issue/128). Change just landed in Tumblweed, openSUSE Leap 15.5 users will receive it in the next few days via a maintenance update. The change will be available also as part of Leap Micro 5.5. Users who don't want to wait can get it from [Base:System/openSUSE-repos](https://build.opensuse.org/package/show/Base:System/openSUSE-repos).
```
Comment (dirkmueller) [inline _posts/2023-07-25-try-out-cdn-with-opensuse-repos.md:28]: Typo Tumbleweed

**Review by dirkmueller — COMMENTED** [review]
I think it would be good to start with the why, rather than explaining the complicated how..

**Comment by lkocman** [comment]
Changes are all merged in, but we need to wait for a new TW snasphot being published.

File: _posts/2023-07-25-try-out-cdn-with-opensuse-repos.md (hunk 1 above)
Comment (lkocman) [reply _posts/2023-07-25-try-out-cdn-with-opensuse-repos.md]: Fixed. Thanks for spotting it.

**Comment by lkocman** [comment]
> 

@dirkmueller do you mean why services,  or why CDN?

**Comment by dirkmueller** [comment]
> @dirkmueller do you mean why services, or why CDN?

why switch to the services package. I mean I kinda don't understand why it isn't the default tbh. what are the upsides, what are the downsides?

The cdn.opensuse.org is still BETA I'd say,  but we can maybe get further feedback on it (and if I understand correctly, we can automatically update users to further improvements if that becomes necessary). 

That said, given that Fastly is  a new premium sponsor of openSUSE I wouldn't mind mentioning them either, but the blog post gets kinda longwinded then..

**Comment by lkocman** [comment]
Added introduction. Thank you Dirk!

**Pull request merged**